src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -256,7 +283,7 @@ class CommandRequestImpl : public CommandImpl,
    * interface that returns response.
    * @return resulting code for sending to mobile application.
    */
-  mobile_apis::Result::eType PrepareResultCodeForResponse(
+  virtual mobile_apis::Result::eType PrepareResultCodeForResponse(
```

Comment:
@AByzhynar done in https://github.com/smartdevicelink/sdl_core/pull/1567/commits/9e2ce859407b0ff871832e4f6e99254cc2eb4a8f